src/commands/debug.js

```diff
@@ -0,0 +1,247 @@
+const URL = require('url').URL
+const chalk = require('chalk')
+const path = require('path')
+
+const { createCompiler } = require('../command-helpers/compiler')
+const { fixParameters } = require('../command-helpers/gluegun')
+const { assertManifestApiVersion, assertGraphTsVersion } = require('../command-helpers/version')
+const { createJsonRpcClient } = require('../command-helpers/jsonrpc')
+const DataSourcesExtractor = require('../command-helpers/data-sources')
+const Protocol = require('../protocols')
+
+const HELP = `
+${chalk.bold('graph debug')} [options] ${chalk.bold('<subgraph-name>')} ${chalk.bold(
+  '[<subgraph-manifest>]',
+)}
+
+Options:
+
+  -e, --endpoint <url>          Graph node GraphQL endpoint synced up to "block" for the debugged subgraph
+  -b, --block <number>          Usually the block at which your subgraph fails
```

Comment:
I would remove the `Usually the`, from my understanding this is the **exact** block at which you want to debug you subgraph.

Perhaps you meant is `The block at which your subgraph usually fails`.

Reply:
I will leave it just `The block at which your subgraph fails.`